crates/sui-core/src/generate_format.rs

```diff
@@ -136,6 +136,9 @@ fn get_registry() -> Result<Registry> {
     let ccd = CheckpointContentsDigest::random();
     tracer.trace_value(&mut samples, &ccd)?;
 
+    let struct_tag = StructTag::from_str("0x2::coin::Coin<0x2::sui::SUI>").unwrap();
+    tracer.trace_value(&mut samples, &struct_tag)?;
```

Comment:
Why is this needed?

Reply:
I don't have the intrinsic reason but upgrading serde from 1.0.152 to any later versions will panic if this is not added.

Reply:
ah I see, got it